internal/gatewayapi/runner/runner.go

```diff
@@ -84,6 +85,10 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 			}
 			// Translate to IR
 			result := t.Translate(&in)
+			yamlXdsIR, _ := yaml.Marshal(&result.XdsIR)
+			r.Logger.Info(string(yamlXdsIR))
+			yamlInfraIR, _ := yaml.Marshal(&result.InfraIR)
+			r.Logger.Info(string(yamlInfraIR))
```

Comment:
This was a surprise to see in the logs. If you want this to stay, can you please update the PR description? If this does stay, you can also add "Fixes #278" and I can close https://github.com/envoyproxy/gateway/pull/279.